.github/workflows/test-env-deploy.yml

```
@@ -233,3 +269,22 @@ jobs:
           group: "UI - Chrome"
           record: true
           tag: Critical, ${{github.event.action}}
+  test-complete:
+    needs: [testmo-report-preparation, cypress-run-selected]
+    if: always()
+    runs-on: ubuntu-latest
```

I would recommend not automatically using the latest available major version as it could suddendly break the workflows.

I recommend setting the major version and upgrading whenever a new LTS stable is added.

```suggestion
    runs-on: ubuntu-22.04
```

---

Fixed